tests/tests.js

```diff
@@ -0,0 +1,256 @@
+        .typeText(Selector('#password'), 'moo')
+        .click(Selector(".form-element.form-button"));
+});
+
+fixture `Acceptance Testing`
+    .page`https://pass.local`;
+
+test('can walk through an nih submission workflow and make a submission - base case', async t => {
+    // Log in
+    await t.useRole(userNih);
+
+    // Go to Submissions
+    const submissionsButton = Selector('.nav-link.ember-view').withExactText('Submissions');
+    await t.expect(submissionsButton.exists).ok();
+    await t.click(submissionsButton);
+
+    var currLocation = ClientFunction(() => window.location.href);
+    await t.expect(currLocation()).eql('https://pass.local/app/submissions');
+
+    // Start new Submission
+    const startNewSubmissionButton = Selector('.ember-view.btn.btn-primary.btn-small.pull-right').withAttribute('href', '/app/submissions/new');
+    await t.expect(startNewSubmissionButton.exists).ok();
+    await t.click(startNewSubmissionButton);
+
+    currLocation = ClientFunction(() => window.location.href);
+    await t.expect(currLocation()).eql('https://pass.local/app/submissions/new/basics');
+
+    // Input DOI
+    const doiInput = Selector('#doi');
+    await t.expect(doiInput().exists).ok();
+    await t.typeText(doiInput, '10.1039/c7an01256j', { paste: true });
+    const toastMessage = Selector('.toast-message').withText("We've pre-populated information from the DOI provided!");
+    await t.expect(toastMessage.exists).ok();
+
+
+    // Check that the title and journal values have been filled in
+    const titleName = Selector('#title');
+    await t.expect(titleName.value)
+        .eql('Quantitative profiling of carbonyl metabolites directly in crude biological extracts using chemoselective tagging and nanoESI-FTMS');
+
+    const journalName = Selector('.form-control').withAttribute('data-test-journal-name-input');
+    await t.expect(journalName.value).eql('The Analyst');
+
+    // Check that the title and journal values cannot be edited
+    await t.typeText(titleName, 'moo');
+    await t.expect(titleName.value)
+        .eql('Quantitative profiling of carbonyl metabolites directly in crude biological extracts using chemoselective tagging and nanoESI-FTMS');
+
+    await t.typeText(journalName, 'moo');
+    await t.expect(journalName.value).eql('The Analyst');
+
+    // Go to Grants
+    const goToGrantsButton = Selector('.btn.btn-primary.pull-right.next');
+    await t.expect(goToGrantsButton.exists).ok();
+    await t.click(goToGrantsButton);
+
+    currLocation = ClientFunction(() => window.location.href);
+    await t.expect(currLocation()).eql('https://pass.local/app/submissions/new/grants');
+
+    // Select a Grant
+    const nihGrant = Selector('#grants-selection-table')
+        .child('table').child('tbody').child('tr').child('td').withExactText('Eye research (07/01/2017 - 06/30/2022)');
+    await t.expect(nihGrant.innerText).eql('Eye research (07/01/2017 - 06/30/2022)');
```

Comment:
I think you might have missed my comment above. Here I would suggest: 
```suggestion
    await t.expect(nihGrant.exists).ok();
```

Reply:
I've changed it from the grant name to the grant award number instead, which also involves changing it to checking for exists as opposed to matching innerText, so I'm not sure how this should apply now.

Reply:
Ah I think I was still looking at the state of the old file. That makes sense and looks good to me.